api/package.json

```diff
@@ -69,9 +69,10 @@
 		"@directus/schema": "file:../packages/schema",
 		"@directus/specs": "file:../packages/specs",
 		"@godaddy/terminus": "^4.4.1",
-		"@slynova/flydrive": "^1.0.3",
-		"@slynova/flydrive-gcs": "^1.0.3",
-		"@slynova/flydrive-s3": "^1.0.3",
+		"@directus/flydrive": "file:../packages/drive/packages/flydrive",
+		"@directus/flydrive-gcs": "file:../packages/drive/packages/flydrive-gcs",
+		"@directus/flydrive-s3": "file:../packages/drive/packages/flydrive-s3",
+		"@directus/flydrive-azureBlob": "file:../packages/drive/packages/flydrive-azureBlob",
```

Comment:
Use lowercase for the package name `azureblob` or `azure-blob`, update references

Reply:
We'll have to "lift" these sub-packages into the "root" packages folder:

`/packages/drive/packages/flydrive-azureBlob` -> `/packages/drive-azure-blob`